mage_ai/__init__.py

```diff
@@ -121,11 +121,10 @@ def clean(
 
 # --------------- Data preparation methods --------------- #
 
-
 def run(pipeline_uuid: str, project_path: str = None, **global_vars) -> None:
     project_path = os.getcwd() if project_path is None else os.path.abspath(project_path)
     sys.path.append(os.path.dirname(project_path))
     pipeline = Pipeline(pipeline_uuid, project_path)
     asyncio.run(
-        pipeline.execute(analyze_outputs=False, global_vars=global_vars, update_status=False)
+        pipeline.execute(analyze_outputs=False, global_vars=global_vars, update_status=True)
```

Comment:
I purposely set it to False because writing to metadata.yaml throws permission error when I run it in Airflow.

Reply:
I see

Reply:
Setting this as false can lead to some errors when running the pipeline if the blocks haven't been executed yet

Reply:
Should it just run all the blocks?

Reply:
yeah it should, but there's some validation that will fail if the status is not set. i'm looking to see if we can skip that validation